cookbooks/aws-parallelcluster-environment/test/controls/spack_spec.rb

```diff
@@ -34,13 +34,13 @@
 
   describe file("#{node['cluster']['spack']['root']}/etc/spack/compilers.yaml") do
     it { should exist }
-    it { should be_owned_by (node['cluster']['spack']['user']).to_s }
-    it { should be_grouped_into (node['cluster']['spack']['user']).to_s }
+    it { should be_owned_by 'root' }
+    it { should be_grouped_into 'root' }
   end
 
   spack = "#{node['cluster']['spack']['root']}/bin/spack"
-  describe "spack commands can run as cluster default user #{node['cluster']['spack']['user']}" do
-    subject { bash("su - #{node['cluster']['spack']['user']} -c '#{spack} find'") }
+  describe "spack commands can run as cluster default user #{node['cluster']['cluster_user']}" do
+    subject { bash("su - #{node['cluster']['cluster_user']} -c 'sudo #{spack} find'") }
```

Comment:
I don't think this needs to be run as sudo does it?

Reply:
Nope it doesnt. Made those changes